build-tools/packages/build-tools/src/common/monoRepo.ts

```diff
@@ -91,6 +91,7 @@ export class MonoRepo {
 		private readonly logger: Logger = defaultLogger,
 	) {
 		this.version = "";
+		this.workspaceGlobs = [];
 		const pnpmWorkspace = path.join(repoPath, "pnpm-workspace.yaml");
 		const lernaPath = path.join(repoPath, "lerna.json");
```

Comment:
Seems like we just use lerna as a task runner these days, so it seems odd that we need to do anything related to lerna in here.

On a related note: Do we actually get any value from using lerna? Can't we replace all use of it with pnpm since pnpm includes task running tools?

Reply:
I'm keeping lerna around for a few reasons. Today it's how fluid-build and its friends decide whether something is a release group root or not. The code is not great, and I am refactoring the release group handling as a whole in a separate PR. The code also uses the lerna version as the release group version, but that's minor and easy to change. Originally I had all these changes together but I realized this piece could be pulled out into its own PR.

The second reason is to hopefully ease experimentation with nx, because lerna v6 switches the task running infra to nx from the previous lerna runner. I remain excited about build caching, and my hope is that this will ease our onboarding or testing of Nx.